bundler.d/puppet.rb

```diff
@@ -3,3 +3,7 @@
   gem 'puppet', '< 5.0.0'
   gem 'ruby-augeas', :require => 'augeas'
 end
+
+group :puppet_proxy_puppet_api do
+  gem 'concurrent-ruby' if RUBY_VERSION > '1.8.7'
```

Comment:
I'd suggest dropping 1.8.7 support entirely here and enable this always; send a PR to foreman-infra to update the test_proxy_develop.yaml file/job and remove it from there.